misc/fibonacci_formula.c

```diff
@@ -0,0 +1,58 @@
+    /**
+     * @file
+     * @brief [finding Fibonacci number of any number n using binet's closed form formula](https://en.wikipedia.org/wiki/Fibonacci_number#Binet's_formula)
+     * @details
+     * compute \f$f_{nth}\f$ Fibonacci number using the binet's formula:
+     * Fn = 1√5 * (1+√5 / 2)^n+1 − 1√5 * (1−√5 / 2)^n+1
+     * @author [GrandSir](https://github.com/GrandSir/)
+    */
```

Comment:
Seems there's extra indentation here.
The description looks good, by the way.
```suggestion
/**
 * @file
 * @brief [finding Fibonacci number of any number n using binet's closed form formula](https://en.wikipedia.org/wiki/Fibonacci_number#Binet's_formula)
 * compute \f$f_{nth}\f$ Fibonacci number using the binet's formula:
 * Fn = 1√5 * (1+√5 / 2)^n+1 − 1√5 * (1−√5 / 2)^n+1
 * @author [GrandSir](https://github.com/GrandSir/)
 */
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant